fdbcli/fdbcli.actor.cpp

```diff
@@ -3177,6 +3162,15 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 		return 1;
 	}
 
+	// Note: refactoring work, will remove the above code finally
+	try {
+		db2 = API->createDatabase(opt.clusterFile.c_str());
+	} catch (Error& e) {
+		fprintf(stderr, "(CAPI)ERROR: %s (%d)\n", e.what(), e.code());
+		printf("(Refactoring): Unable to connect to cluster from `%s'\n", ccf->getFilename().c_str());
```

Comment:
What is the ultimate intent here? That this error string will look exactly like the one above at line 3160 and 3161 once we're only using the one DB object? In other words, `(CAPI)` and `(Refactoring)` will be removed?

Reply:
Yeah, please ignore this.
Removed those unnecessary comments.